dashboard/test/ui/features/platform/one_trust.feature

```diff
@@ -24,9 +23,9 @@ Feature: OneTrust integration
 
   Scenario: OneTrust cookie pop-up shows when self-hosting OneTrust libraries on code.org
     Given I create a student named "Alice"
-    Given I am on "http://studio.code.org/home?otreset=false&otgeo=gb"
+    Given I am on "http://studio.code.org/home?otreset=true&otgeo=gb"
```

Comment:
Was missing this in my original reverted PR, my new one adds this change: https://github.com/code-dot-org/code-dot-org/pull/61000